cmd/opm/alpha/bundle/build.go

```diff
@@ -76,12 +76,17 @@ func newBundleBuildCmd() *cobra.Command {
 	bundleBuildCmd.Flags().StringVarP(&outputDirArgs, "output-dir", "u", "",
 		"Optional output directory for operator manifests")
 
+	bundleBuildCmd.Flags().BoolVarP(&skipValidationArgs, "skip-validation", "v", false,
```

Comment:
are you saying we never want to skip validation? Because that would say we are completely trusting validation and believe that there is no edge case where the users want to disable validation and can not do so.

Reply:
yes -- our validation should never be incorrect, if it is, there's a bug

Reply:
i will commit this as an individual commit.